pages/RapPage.js

```diff
@@ -30,31 +30,49 @@ export default function RapPage() {
   return (
     <>
       <Header />
-      <StyledDiv>
+      <StyledHeadDiv>
         <BackButton />
         <ShoppingCart />
-      </StyledDiv>
-      <div>
-        <h2>RAP</h2>
-      </div>
+      </StyledHeadDiv>
+      <StyledRapLineDiv>
+        <StyledRapP>RAP</StyledRapP>
+        <StyledHr />
+      </StyledRapLineDiv>
       <div>
         <ul>
           {data.map((rapstars) => (
-            <li key={rapstars._id}>
-              <Image
+            <StyledLi key={rapstars._id}>
+              <StyledImage
                 src={rapstars.image}
-                width={"280"}
-                height={"150"}
+                width={"550"}
+                height={"300"}
                 alt="Rap-Artist"
+                priority="high"
+                style={{
+                  position: "relative",
+                  zIndex: -1,
+                }}
```

Comment:
Try to avoid inline styling. You already styled the image, why not include the `position` and `z-index` as well?